torch/nn/functional.py

```diff
@@ -1829,6 +1829,24 @@ def hardsigmoid(input: Tensor, inplace: bool = False) -> Tensor:
     return torch._C._nn.hardsigmoid(input)
 
 
+def bias(input: Tensor, bias_value: Tensor) -> Tensor:
+    r"""
+    Adds the bias to the incoming data.
+
+    This operator supports :ref:`TensorFloat32<tf32_on_ampere>`.
+
+    Shape:
+
+        - Input: :math:`(N, *, num\_features)` N is the batch size, `*` means any number of
+          additional dimensions
+        - Bias: :math:`(num\_features)`
+        - Output: :math:`(N, *, num\_features)`
```

Comment:
nit: shapes here should match documented shapes for the module form.

I personally like `(*, num_features), where * is any number of dimensions, including none` better because I think it's more accurate, as e.g. `torch.randn(num_features)` should be a valid input. When the `N` is present, it makes it seem required

Reply:
looks good to me!